### PR TITLE
Database schema cleanup

### PR DESCRIPTION
**Pull request by murrant**
Remove several inconsistencies (mostly with data types and lengths)
Make things a bit nicer to implement with Laravel database migrations.
poller* tables had f'd up indexes
removed an unused field with screwy definition

probably needs to be tested on an older version of mysql/mariadb to make sure.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: misc/db_schema.yaml [hunk 1]
```diff
@@ -1615,20 +1608,20 @@ users:
     - { Field: email, Type: varchar(64), 'Null': false, Extra: '' }
     - { Field: descr, Type: char(30), 'Null': false, Extra: '' }
     - { Field: level, Type: tinyint(4), 'Null': false, Extra: '', Default: '0' }
-    - { Field: can_modify_passwd, Type: tinyint(4), 'Null': false, Extra: '', Default: '1' }
-    - { Field: created_at, Type: timestamp, 'Null': false, Extra: '', Default: '1970-01-02 00:00:01' }
+    - { Field: can_modify_passwd, Type: tinyint(1), 'Null': false, Extra: '', Default: '1' }
+    - { Field: created_at, Type: timestamp, 'Null': false, Extra: '', Default: CURRENT_TIMESTAMP }
```
Comment (laf) [inline misc/db_schema.yaml]: I'll need to check this but I'm sure you can't have two CURRENT_TIMESTAMPs in one table with strict mode on

Comment (murrant) [reply misc/db_schema.yaml]: Could be, I just checked and I thought I was running in strict, but I was not.

Comment (laf) [reply misc/db_schema.yaml]:
```
260 -> 261 ...SQL Error! SQLSTATE[HY000]: General error: 1293 Incorrect table definition; there can be only one TIMESTAMP column with CURRENT_TIMESTAMP in DEFAULT or ON UPDATE clause (SQL: ALTER TABLE `users` CHANGE `created_at` `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ;)
  /opt/librenms/includes/sql-schema/update.php:92
```

Comment (murrant) [reply misc/db_schema.yaml]: I think only one could have the `on update CURRENT_TIMESTAMP` right?  default CURRENT_TIMESTAMP is ok?

Comment (laf) [reply misc/db_schema.yaml]: No it says you can't for default in that output

**Comment by laf** [comment]
Leave the rebase on this now but I plan to merge this in Monday.

**Comment by laf** [comment]
Rebased to try and get this out this morning but it's failing tests and I don't have any time to check into it.

**Pull request merged**